tests/test_unstable/test_configuration.py

```diff
@@ -212,3 +213,17 @@ def test_from_env() -> None:
 
     # Check that the produces cogniteclient object is valid
     assert len(client.assets.list(limit=1)) == 1
+
+
+def test_parse_file_size() -> None:
+    assert FileSizeConfig(expression="154584").bytes == 154584
```

Comment:
This just won't work...

Remember that this is supposed to be used from YAML. Write a test that loads this configuration from YAML, like

```
my_config:
    file_size: 10GB
    other_file_size: 4kB
```

Reply:
I don't see why you would have to switch to a helper method.